db/migrate/20240927200908_add_archived_at_to_form.rb

```diff
@@ -0,0 +1,23 @@
+class AddArchivedAtToForm < ActiveRecord::Migration[7.1]
+  def up
+    add_column :forms, :archived_at, :datetime
+
+    execute <<-SQL.squish
+      WITH most_recent_form_archives AS (
```

Comment:
This update executed in ~3 seconds in my production-sized local version of the db.